src/main/java/seedu/model/doctorappointment/AppointmentList.java

```diff
@@ -36,32 +46,80 @@ public static void addAppointment(String[] inputArray) throws IOException {
     public static void listAppointment(String input) throws HealthVaultException, EmptyListException, ParseException {
 
         String indicator = "A";
+        String doctorID;
         String[] inputArray = input.split("");
         String iD = inputArray[0];
         if (appointmentList.size() == 0) throw new EmptyListException();
         else {
-            if (iD.equals("A")) {
+            if (input.equals("all")) {
+                doctorFound.clear();
+                indicator = "all";
+                DoctorAppointmentUI.AptPrintList(indicator);
+                UI.showLine();
+                for (int i = 0; i < appointmentList.size(); i++) {
+                    DoctorAppointmentUI.printList(appointmentList.get(i), indicator);
+
+                    /*extraList.clear();
```

Comment:
Just checked the latest version and this commented code is still there. Please remove it bah if not used.